graphql/schema.graphql

```diff
@@ -2,4 +2,6 @@ type Query {
   suggestAddresses(searchTerm: String!): [AddressSuggestion!]!
 
   getAddressByExternalId(id: String!): Address!
+
+  getLogo: Image!
```

Comment:
just a suggestion, can we rename this to `providerLogo` or something?

Reply:
The query to `getProviderLogo` or the type to `ProviderLogo`? If you meant the query, I agree

Reply:
the query to `providerLogo`, without the `get` prefix

Reply:
Kay.

Any opinions on the other queries as well? Recently I noticed that it's not very common to name queries like functions, but at the same time, I found no nomenclature guidelines or something like that.

Reply:
I'd rename the `getAddressByExternalId` as well, but that would be a breaking change right? I like to name the queries after the "object" I'm querying for (for example, if we had a query to get the current logged user I'd name it as something like `me`, `viewer` or `currentUser`), so for the `getByExternalId` I'd rename it as `address` and add the `externalId` as a parameter (like a filter).

I don't think there is any nomenclature guide 😂 but I follow something in the lines above